Tests/SourceKitLSPTests/LocalSwiftTests.swift

```diff
@@ -1510,6 +1504,9 @@ final class LocalSwiftTests: XCTestCase {
       log("Received diagnostics for open - semantic")
     })
 
+    // Send a request that triggers a syntax tree to be built.
+    _ = try sk.sendSync(FoldingRangeRequest(textDocument: .init(uri)))
```

Comment:
What's this needed for?

Reply:
Previously, we were always eagerly computing the syntax tree whenever we opened a file. Now, we only compute it once the first request requires it (and then implicitly keep it up to date on every edit, the thought being that you’ll probably need it again and doing incremental parses is faster than re-parsing from scratch when it’s needed next). 

To test that we are parsing incrementally on the next edit, we need to trigger that initial build of a syntax tree, which is what I’m doing here.